src/components/cards/LibraryGameCard.vue

```diff
@@ -24,29 +31,41 @@
       </template>
 
       <template v-slot:metadata>
-        <b-icon-person-fill class="text-muted mr-2"></b-icon-person-fill>
-        <span class=" text-muted">{{
-          num_players(game.game.min_players, game.game.max_players)
-        }}</span>
-        <b-icon-clock-fill
-          class="ml-4 mr-2 text-muted"
-          font-scale="0.8"
-        ></b-icon-clock-fill>
-        <span class=" text-muted"
-          >{{ playtime(game.game.min_playtime, game.game.max_playtime) }}
-        </span>
+        <div v-if="!$store.getters['users/current'].is_staff">
```

Comment:
Suggestion: it's more readable if the `if` is not negated (the content must be swapped with the else)
```suggestion
        <div v-if="$store.getters['users/current'].is_staff">
```